plugins/package-managers/conan/src/main/kotlin/Conan.kt

```diff
@@ -56,6 +56,7 @@ import org.ossreviewtoolkit.plugins.api.OrtPlugin
 import org.ossreviewtoolkit.plugins.api.OrtPluginOption
 import org.ossreviewtoolkit.plugins.api.PluginDescriptor
 import org.ossreviewtoolkit.utils.common.CommandLineTool
+import org.ossreviewtoolkit.utils.common.alsoIfNull
```

Comment:
Esp. with this commit, I don't really see why one would need to disable the Bazel <-> Conan interaction.